react-native-in-app-payments-quickstart/app/service/ChargeCustomerCard.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable camelcase */
+/*
+ Copyright 2019 Square Inc.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+import { CHARGE_CUSTOMER_CARD_SERVER_URL } from '../Constants';
+import ChargeCustomerCardError from '../ChargeCustomerCardError';
+
+export default async function createCustomerCard(customer_id, customer_card_id) {
```

Comment:
Should the function here be `chargeCustomerCard`?

Reply:
Yes, will update.